File: Modul-2/PRAK205/PRAK205-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>
#include <math.h>

int main(){
    int A,B;
    scanf("%d %d",&A,&B);
    int Alas=sqrt(B*B-A*A);
    int keliling=Alas+A+B,luas=Alas*A*1/2;
    printf("Alas = %d cm\n",Alas);
    printf("Tinggi = %d cm\n",A);
    printf("Keliling = %d cm\n",keliling);
    printf("Luas = %d cm^2",luas);
    return 0;
}

File: Modul-2/PRAK205/PRAK205-2310817210010-Harry-Pratama-Yunus.py
import math
Angka=input().split()
if len(Angka)==1:
    A=int(Angka[0])
    B=int(input())
elif len(Angka)==2:
    A,B=map(int,Angka)
Alas=round(math.sqrt(B*B-A*A))
keliling=round(Alas+A+B)
luas=round(Alas*A*1/2)
print("Alas = " + str(Alas)+" cm")
print("Tinggi = "+ str(A)+" cm")
print("Keliling = "+ str(keliling)+" cm")
print("Luas = "+ str(luas)+" cm^2")